### ISSUE TITLE
Displaying boost::geometry::index::rtree in Geometry Watch

### ISSUE THREAD

**Issue by kuchenluege** [awulkiew/graphical-debugging#55]
![grafik](https://user-images.githubusercontent.com/71273405/222423486-6c89d0db-cbcf-462b-a1e8-8c05393c8b29.png)
When I insert points into an rtree, the points can be displayed but the rtree can't? Is this a bug or I am I using the tool wrong?

**Comment by awulkiew** [comment]
When the rtree is visualized only the contained elements, e.g. points are displayed. The internal structure of the rtree, i.e. bounding boxes of nodes are not visualized. In other words it's treated like a container.

Or did I misunderstood the problem and the points stored in the rtree are not visualized for you at all?

**Comment by kuchenluege** [comment]
![grafik](https://user-images.githubusercontent.com/71273405/223671243-ba5cafc9-cb44-44d8-8668-6425d0969ae0.png)
No, nothing in the rtree is being displayed at all.

```
#include <boost/geometry.hpp>
#include <boost/geometry/index/rtree.hpp>

using Point = boost::geometry::model::point<float, 2, boost::geometry::cs::cartesian>;
using RTree = boost::geometry::index::rtree<Point, boost::geometry::index::rstar<4>>;

int main()
{
    RTree rt;

    Point center(0.0, 0.0);
    int const LEN = 10;
    Point points[LEN];
    for (Point& pt : points) {
        pt.set<0>(center.get<0>() + (float)rand() / 100);
        pt.set<1>(center.get<1>() + (float)rand() / 100);
        rt.insert(pt);
    }
    return 0;
}
```

**Comment by awulkiew** [comment]
Ok, thanks for the info, I'll check it out.

**Comment by awulkiew** [comment]
Fixed in v0.34.
